spring-cloud-dataflow-admin/src/main/java/org/springframework/cloud/dataflow/admin/completion/TapOnChannelExpansionStrategy.java

```diff
@@ -47,13 +47,12 @@ public boolean shouldTrigger(String dslStart, Exception exception) {
 
 	@Override
 	public void addProposals(String dsl, ParseException exception, int detailLevel, List<CompletionProposal> collector) {
-		String streamAndModule = dsl.substring("tap:stream:".length());
-		int firstDot = streamAndModule.indexOf('.');
-		String streamName = streamAndModule;
+		String streamName = dsl.substring("tap:stream:".length());
 		String moduleName = "";
-		if (firstDot > 0) {
-			streamName = streamAndModule.substring(0, firstDot);
-			moduleName = streamAndModule.substring(firstDot + 1);
+		if (streamName.contains(".")) {
+			String[] splited = streamName.split("\\.")
```

Comment:
Thanks for the Pull Request. I would probably change "splited" (mis-spelled and/or not a common use of the adjective, which would normally be just "split" itself) to either "splits" or "tokens".